zdoom.vcproj

```diff
@@ -6704,6 +6704,18 @@
 				>
 			</File>
 		</Filter>
+		<Filter
+			Name="Portals"
+			>
+			<File
+				RelativePath=".\src\portal.cpp"

```

Comment:
Got somehow lost during the merge with master. Put it back. &updated the lists as well.